languages/kotlin/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,47 @@
+Owner of your local pizza place asked you to help her to count cooking time for different pizzas. In this exercise you're going to write few functions to calculate that.
+
+You have three tasks.
+
+## 1. Calculate remaining baking time in minutes
+
+Define and implement `remainingMinutesInOven()` function that will count how many more minutes you need to keep pizza in the oven.
+
+Function should accept one **optional** parameter (with default value) for the amount of minutes that passed since you've started baking pizza. If parameter is missing - assume that you've just put pizza into the oven and **0 (zero)** minutes passed already.
+
+For the simplicity - let's assume that all types of pizzas requires exactly 40 minutes in the oven to be ready.
+
+```kotlin
+remainingMinutesInOven()   // => 40
+
+remainingMinutesInOven(10)   // => 30
+```
+
+## 2. Calculate the preparation time in minutes
+
+Before baking pizza you need to prepare dough, roll it out and put toppings onto it.
+
+Define and implement `preparationTimeInMinutes()` function that will count how many minutes it will take to prepare pizza before putting it into the oven.
```

Comment:
```suggestion
Define and implement the `preparationTimeInMinutes()` function that will count how many minutes it will take to prepare pizza before putting it into the oven.
```